rclc/src/rclc/executor.c

```diff
@@ -1112,6 +1112,7 @@ bool rclc_executor_trigger_one(rclc_executor_handle_t * handles, unsigned int si
           case SUBSCRIPTION:
             if (handles[i].subscription == obj) {
               return true;
+            } else {
```

Comment:
Why this empty else?

Reply:
This was an artifact from debugging. The else-statement contained a printf, which I removed. I forgot to remove the else-statement. This is resolved now.